apps/web/components/Header/Header.tsx

```diff
@@ -89,7 +89,10 @@ export const Header: FC<HeaderProps> = ({
                     )}
                     <Hidden below="lg">
                       <Box marginLeft={marginLeft}>
-                        <Link href="//minarsidur.island.is/" skipTab>
+                        <Link
+                          href={english ? '/en/login' : '/innskraning'}
```

Comment:
Consider adding this url to the link resolver to keep internal link resolution consistent

Reply:
Yes! Makes sense.

Reply:
@stefanhar - this has been fixed. Any chance of a re-review when you got the time 🙏  😃